app/views/deploys/_checklist.html.erb

```diff
@@ -6,8 +6,8 @@
   <ul class="deploy-checklist">
     <%- stack.checklist.each do |check| -%>
       <li class="deploy-checklist__item">
+        <%= check_box_tag :checklist, nil, false, class: 'deploy-checklist__item__checkbox' %>
         <label class="deploy-checklist__item__label">
-          <%= check_box_tag :checklist, nil, false, class: 'deploy-checklist__item__checkbox' %>
```

Comment:
Correct me if I'm wrong, but this change won't allow you to click on the text to check the box anymore right?

Should we add an `id` and a `for` attribute?